src/components/Database/Database.tsx

```diff
@@ -73,7 +71,7 @@ export const Database: React.FC<Props> = ({ config, namespace }) => {
         <>
           <InteractiveBreadCrumbBar
             base={urlBase}
-            path={new URL(ref.toString()).pathname}
+            path={path || '/'}
```

Comment:
Does `path` also need a trailing slash here?

Reply:
No, trailing slashes are actually bad in the breadcrumbs. I will add tests in another PR. For now this gets deep linking working.